### PR TITLE
Login Page

### PR DESCRIPTION
**Pull request by Willo0ow**
- add Login Page
- fix navbar links to login/register
- add logout api
- show logout button  instead  of lgoin&register in navbar
- handle taken email error while register and incorrect credentials while login
- add notifications
Zrzut ekranu 2022-09-12 o 10 40 08
clode #16

### CODE REVIEW

File: src/api/useAuthApi.js [hunk 1]
```diff
@@ -1,12 +1,49 @@
 import axios from "@/axios";
 import { auth } from "@/api/endpoints.js";
+import { notify } from "@kyvg/vue3-notification";
 
 export const saveRegisterUser = async (registerData) => {
   try {
     const res = await axios.post(`/${auth.REGISTER}`, registerData);
     return { user: res.data, token: res.headers.authorization };
+  } catch (error) {
+    if (error.response.data.errors?.email[0] === "has already been taken") {
```
Comment (elmir-ismayilzada) [inline src/api/useAuthApi.js]: Here' it would be good to use `.includes` instead of strict compare

File: src/stores/auth.js [hunk 2]
```diff
@@ -11,7 +12,7 @@ export const useAuthStore = defineStore({
   }),
   getters: {
     isUserLoggedIn() {
-      return !!this.token;
+      return this.token ? true : false;
```
Comment (elmir-ismayilzada) [inline src/stores/auth.js]: Why did you remove that? Also you can use `return Boolean(this.token)`

File: src/pages/LoginPage.vue [hunk 3]
```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="login">
+    <div class="login__title">
+      <SectionTitle variation="40-60-80">Hi there!</SectionTitle>
+      <SectionTitle color="bombay" variation="40-60-80"
+        >Care to log in?</SectionTitle
+      >
+    </div>
+    <FormCard @submit="handleSubmit">
```
Comment (elmir-ismayilzada) [inline src/pages/LoginPage.vue:9]: You can use here `@submit.prevent` and there's no need to call `event.preventDefault()` in `handleSubmit` method

Comment (Willo0ow) [reply src/pages/LoginPage.vue:9]: You're right. Now when I looked at it I realized, I don't have to do it at all, because it's already prevented in FormCard component: 
`    <form
      class="form-card__form"
      @submit.prevent="$emit('submit', $event)"`

File: src/stores/auth.js (hunk 2 above)
Comment (Willo0ow) [reply src/stores/auth.js]: I guess it felt more explicit. But I like your suggestion with Boolean()

**Review by elmir-ismayilzada — APPROVED** [review]
